### PR TITLE
#MAR-1907 add area of expertise

### PR DESCRIPTION
**Pull request by malmas18**
there is also a fix for contract length where you cannot type more than 100 characters

### CODE REVIEW

**Review by matt-sm — APPROVED** [review]

File: frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml [hunk 1]
```diff
@@ -39,9 +39,17 @@ options:
     label: "Content and publishing"
     value: "Content and Publishing"
   -
-    description: "Change, training and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice, agile coaching, change management and training in new ways of working."
+    description: "Change, training and transformation will soon be removed. New sellers should consider offering Change and Transformation and/or Training, Learning and Capability."
```
Comment (tobiasbrown) [inline frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml]: `Training, Learning and Capability` or `Training, Learning and Development`?

File: frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml [hunk 2]
```diff
@@ -39,9 +39,17 @@ options:
...
     label: "Change, training and transformation"
     value: "Change, Training and Transformation"
+  -
+    description: "Change and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice, change management and transitioning to new ways of working."
+    label: "Change and transformation"
+    value: "Change and Transformation"
+  -
+    description: "Training, learning and development builds skills and expertise to deliver government services digitally. Examples include training needs analysis, developing and delivering learning solutions, coaching and mentoring and learning impact evaluation."
+    label: "Training, learning and development"
+    value: "Training, Learning and Development"
```
Comment (tobiasbrown) [inline frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml:52]: `Training, Learning and Capability` or `Training, Learning and Development`?

File: schemas/briefs-digital-marketplace-digital-professionals.json [hunk 3]
```diff
@@ -21,6 +21,8 @@
         "Support and Operations",
         "Content and Publishing",
         "Change, Training and Transformation",
+        "Change and Transformation",
+        "Training, Learning and Development",
```
Comment (tobiasbrown) [inline schemas/briefs-digital-marketplace-digital-professionals.json:25]: `Training, Learning and Capability` or `Training, Learning and Development`?

File: frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml [hunk 4]
```diff
@@ -39,15 +39,15 @@ options:
...
   -
-    description: "Change and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice and change management."
+    description: "Change and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice, change management and transitioning to new ways of working."
     label: "Change and transformation"
     value: "Change and Transformation"
   -
-    description: "Training, learning and capability."
+    description: "Training, learning and capability builds skills and expertise to deliver government services digitally. Examples include training needs analysis, developing and delivering learning solutions, coaching and mentoring and learning impact evaluation."
     label: "Training, learning and capability"
     value: "Training, Learning and Capability"
```
Comment (tobiasbrown) [inline frameworks/digital-marketplace/questions/briefs/areaOfExpertiseSpecialists.yml]: `Training, Learning and Capability` or `Training, Learning and Development`?

**Pull request merged**